acceptance-tests/src/test-support/java/tech/pegasys/artemis/test/acceptance/dsl/data/FinalizedCheckpoint.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.test.acceptance.dsl.data;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.primitives.UnsignedLong;
+import com.google.gson.annotations.SerializedName;
+import java.util.Objects;
+import org.apache.tuweni.bytes.Bytes32;
+
+public class FinalizedCheckpoint {
+
+  private final UnsignedLong epoch;
+
+  @SerializedName("root")
```

Comment:
We don't need `@SerializedName` here as it matches the actual name.

Reply:
removed.